platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -46,6 +46,11 @@ The `Settings.getDefaults()` incubating method introduced in 8.10 has been remov
 Use`Settings.defaults(Action<SharedModelDefaults>)` method, taking a lambda, instead.
 The change is driven by the need to interpret such default values in the scope of individual projects and not at Settings level.
 
+==== Upgrade to Guava 33.4.5
+
+Guava was updated from 32.1.2 to 33.4.5, which deprecates some core features like `Charsets` or `Ints.compare`.
```

Comment:
🤔 It would be nice to note the replacement types in the JDK here, unless guava's own docs already explain these at the link below.

Reply:
They're explained on the deprecation note, I don't think we need to do anything specific.